examples/sqlite.rs

```diff
@@ -91,20 +91,26 @@ fn name_to_type(name: &str) -> PgWireResult<Type> {
 }
 
 fn row_desc_from_stmt(stmt: &Statement, format: &Format) -> PgWireResult<Vec<FieldInfo>> {
-    stmt.columns()
-        .iter()
-        .enumerate()
-        .map(|(idx, col)| {
-            let field_type = name_to_type(col.decl_type().unwrap())?;
-            Ok(FieldInfo::new(
-                col.name().to_owned(),
+    let columns = stmt.columns();
+    let mut fields = Vec::with_capacity(columns.len());
+
+    for (idx, column) in columns.iter().enumerate() {
+        if let Some(decl_type) = column.decl_type() {
+            let field_type = name_to_type(decl_type)?;
+
+            fields.push(FieldInfo::new(
+                column.name().to_owned(),
                 None,
                 None,
                 field_type,
                 format.format_for(idx),
             ))
-        })
-        .collect()
+        } else {
+            return Ok(vec![]);
```

Comment:
Sorry for back and force. Do you think if there is any better solution for columns without decl_type? 

I will investigate on this too. Perhaps we can use postgres' `Unknown` type here and return the text representation of this field.